man/generate.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+#
+# Generate man pages for docker/docker
+#
+
+set -eu
+
+mkdir -p ./man/man1
+
+MD2MAN_REPO=github.com/cpuguy83/go-md2man
+MD2MAN_COMMIT=a65d4d2de4d5f7c74868dfa9b202a3c8be315aaa
+
+( 
+	go get -d "$MD2MAN_REPO"
+	cd "$GOPATH"/src/"$MD2MAN_REPO"
+	git checkout "$MD2MAN_COMMIT" &> /dev/null
+	go install "$MD2MAN_REPO"
+)
+
+VENDOR_MD5="$(md5sum vendor.conf)"
+cp vendor.conf /tmp/vendor.conf
+cp man/vendor.tmp vendor.conf
+
+grep -v '^#' man/vendor.tmp | while read dep; do
+	vndr $(echo "$dep" | cut -d' ' -f1)
+done 
```

Comment:
There has to be a better way to do this.

Why not keep this as a separate Dockerfile? Now that we have ARG in FROM I think we can have a single file.